docs/source/languagesections/__init__.py

```diff
@@ -36,11 +39,11 @@ def add_assets(app):
 def copy_assets(app, exception):
     if app.builder.name != 'html' or exception:
         return
-    app.info('Copying examplecode stylesheet/javascript... ', nonl=True)
```

Comment:
`app.info` was deprecated in Sphinx 1.8.x and has been removed in Sphinx 2.x. It is recommended that the Sphinx `logging` module be used instead. These changes make our docs build process compatible with Sphinx 2.x